modules/apigee_edge_teams/src/Entity/TeamMemberRole.php

```diff
@@ -56,6 +60,31 @@ final class TeamMemberRole extends ContentEntityBase implements TeamMemberRoleIn
 
   use EntityChangedTrait;
 
+  /**
+   * {@inheritdoc}
+   */
+  public function access($operation, AccountInterface $account = NULL, $return_as_object = FALSE) {
+    if (!$account) {
+      // If for we have hit this without an account return forbidden.
+      return AccessResult::forbidden();
+    }
+
+    $result = AccessResult::allowedIfHasPermissions($account, [
+      'administer team',
+      'manage team members',
+    ], 'OR');
+
+    if ($result->isNeutral()) {
+      $team = $this->getTeam();
+      $team_permission_handler = \Drupal::service('apigee_edge_teams.team_permissions');
+      $result = AccessResultAllowed::allowedIf(in_array('team_manage_members', $team_permission_handler->getDeveloperPermissionsByTeam($team, $account)))
```

Comment:
The $result created in line 72 should be concatenated with this new one by using or to preserve cacheability information originating from that result object.

Reply:
Hi @audave,
I agree with @mxr576 , 
Could you merge the `$result` on line 72 with the new one, so we can review this PR and approve it.

Also, could you please verify your CLA which is failing.

Reply:
allowedIf method does not take a conjunction.

The method used here matches the implementation in TeamAppListByTeamAccess->access()

Reply:
@audave 
Could you please sign the Google cla.
Ref [here](https://github.com/apigee/apigee-edge-drupal/pull/848/checks)

Reply:
@audave 
Could you try to concatenate like this
```
return AccessResult::allowedIfHasPermissions($account, [
    'administer team',
    'manage team members',
    ], 'OR')
    ->orIf(AccessResultAllowed::allowedIf(in_array('team_manage_members', $team_permission_handler->getDeveloperPermissionsByTeam($team, $account))))
    ->addCacheableDependency($team)
    ->cachePerUser();
```

Reply:
@audave We will create separate PR for above code change.